docs/src/reference/configuration.rst

```diff
@@ -141,410 +168,513 @@ title
+-  examples:
 
-filter
-------
+   -  ``--max-date 2021-04-01``
+   -  ``--max-date 2021.25``
+
+priorities
+~~~~~~~~~~
 
 -  type: object
--  description: Filters to apply to strain metadata and sequences prior to subsampling and tree inference. The workflow applies an implicit filter on the maximum collection dates later than today.
+-  description: Parameters to prioritize strains selected for the current subsampling rule. Currently, the workflow supports two ``type``\ s of priority, ``proximity`` and ``file``.
+-  description [proximity]: ``proximity`` selects samples that are genetically similar to the ``focus`` sample set; the ``focus`` sample set must be a rule in the current subsampling scheme.
+-  example [proximity]:
 
-.. contents::
-   :local:
+.. code:: yaml
 
-min_length
-~~~~~~~~~~
+   subsampling:
+     my-scheme:
+       my-first-rule:
+         max_sequences: 10
+       my-second-rule:
+         max_sequences: 10
+         # Prioritize sequences that are genetically similar to
+         # sequences in the sequences selected by the
+         # `my-first-rule` rule.
+         priorities:
+           type: proximity
+           focus: my-first-rule
 
--  type: integer
--  description: Minimum number of valid nucleotides (A, C, T, or G) for a genome to be included in the analysis by ``augur filter --min-length``.
--  default: ``27000``
+-  description [file]: ``file`` selects samples based on arbitrarily-defined rankings in a TSV file formatted as ``strain\tnumber``. The numbers are only used to sort the samples, and are therefore arbitrary. Higher values = higher priority.
 
-exclude_where
-~~~~~~~~~~~~~
+-  example [file]:
 
--  type: string
--  description: Conditional tests of metadata columns used to exclude strains from the analysis by ``augur filter --exclude-where``
--  default: ``"division='USA'"``
+.. code:: yaml
 
-exclude_ambiguous_dates_by
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+   subsampling:
+     my-scheme:
+       my-first-rule:
+         max_sequences: 10
+         group_by: "country"
+         priorities:
+           type: "file"
+           file: "path/to/priorities.tsv"
 
--  type: string
--  description: Level date ambiguity used to exclude strains from the analysis by ``augur filter --exclude-ambiguous-dates-by``
--  default: ``any``
--  examples:
+::
 
-   -  ``any``
-   -  ``day``
-   -  ``month``
-   -  ``year``
+   hCoV-19/USA/CZB-1234/2021   8.2
+   hCoV-19/USA/CZB-2345/2021   0
+   hCoV-19/USA/CZB-3456/2021   -3.1
 
-min_date
-~~~~~~~~
 
--  type: float or string
--  description: Minimum collection date for strains to include in the analysis used by ``augur filter --min-date``. Dates can be numeric floating point values (e.g., ``2019.74``) or ISO 8601-style strings (e.g., ``2019-10-01``).
--  default: ``2019.74``
+Cross-Workflow configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Could you add a short description here (and other `<h2>`s) like you did for **Main Workflow configuration**? I'm not sure what "Cross-Workflow" means.

Reply:
For sure! [64ab92c](https://github.com/nextstrain/ncov/pull/907/commits/64ab92c9733c13f6e4020334609c278447f2cf91)